core/src/org/javarosa/core/util/DataUtil.java

```diff
@@ -15,6 +15,12 @@
     static final  int high = 400;
     static Integer[] iarray;
     
+    static UnionLambda unionLambda;
+    
+    { 
+        unionLambda = new UnionLambda();
```

Comment:
Oh, sorry, this is a static initializer (it's actually unrelated to the declaration above), but it occurs to me that it should actually just be on the static declration. Will fix.